releasenotes/notes/fix-optimize-swap-before-measure-67e8896da2215d49.yaml

```
@@ -3,8 +3,8 @@ fixes:
   - |
     Fixed an issue with the :class:`~.OptimizeSwapBeforeMeasure` pass where
     it would incorrectly optimize circuits involving swap and measure 
-    instructions. This commit fixes the bug by changing `DAGCircuit.successors`
-    to `DAGCircuit.descendants`. Also, added a couple of extra tests to ensure 
+    instructions. This commit fixes the bug by changing ``DAGCircuit.successors``
+    to ``DAGCircuit.descendants``. Also, added a couple of extra tests to ensure 
```

2. This is me being nitpicky but it's always nicer to se API docs links (in this case, links for methods):
```suggestion
    instructions. This commit fixes the bug by changing :meth:`.DAGCircuit.successors`
    to :meth:`.DAGCircuit.descendants`. Also, added a couple of extra tests to ensure 
```